### ISSUE TITLE
Add Multi-Upload/Download Options

### ISSUE THREAD

**Issue by joe-batt** [fredldotme/harbour-owncloud#40]
Unless I am missing something, it is currently only possible to download and upload individual files. For a larger number this can be pretty annoying and I usually resort to other means like the web interface.
Would in be possible to add a file multiselect or download/upload options for directories?

**Comment by fredldotme** [comment]
Which platform are you using GhostCloud on?

**Comment by joe-batt** [comment]
I'm using it on Sailfish OS 3.2.

**Comment by fredldotme** [comment]
Upload on SailfishOS should already be multi-select. There's no option yet for multi-download but patches welcome. The current file downloading machinery already supports that use case, it's just not hooked up to the UI.

**Comment by joe-batt** [comment]
You are right multi-upload is possible via share with the additionals installed. 

It's good to know that multi-download is basically implemented, I haven't checked the code, yet. If I find the time I'll see I can create a patch.
